<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallProperties</class>
 <widget class="QWidget" name="CallProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>197</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="DataComboBox" name="type"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelProgram">
       <property name="text">
        <string>Program</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ReferenceComboBox" name="program"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPrimitives">
       <property name="text">
        <string>Primitives</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ReferenceComboBox" name="primitives">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelFramebuffer">
       <property name="text">
        <string>Framebuffer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="ReferenceComboBox" name="framebuffer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelGroupsX">
       <property name="text">
        <string>Groups X</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="numGroupsX">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelGroupsY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="numGroupsY">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelGroupsZ">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="numGroupsZ">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65536</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReferenceComboBox</class>
   <extends>QComboBox</extends>
   <header>session/ReferenceComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>DataComboBox</class>
   <extends>QComboBox</extends>
   <header>session/DataComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
